motioneye/motionctl.py

```diff
@@ -69,6 +69,21 @@ def find_motion():
 
     return _motion_binary_cache
 
+def find_libcamerify():
+
+    if settings.LIBCAMERIFY:
+
+        if os.path.exists(settings.LIBCAMERIFY):
```

Comment:
Just to align coding:
```suggestion
def find_libcamerify():
    if settings.LIBCAMERIFY:
        if os.path.exists(settings.LIBCAMERIFY):
```